Description: Drop some examples, which require Qt
 It is neccessary for the fixing of arm-compilation
 where Qt is not available
Author: Anton Gladky <gladk@debian.org>
Bug-Debian: https://bugs.debian.org/793304
Last-Update: 2015-07-29

--- vtk6-6.2.0+dfsg1.orig/Examples/Infovis/Cxx/CMakeLists.txt
+++ vtk6-6.2.0+dfsg1/Examples/Infovis/Cxx/CMakeLists.txt
@@ -26,11 +26,6 @@ if(vtkViewsInfovis_LOADED)
 endif()
 
 # If Qt is enabled include the QT based examples
-if(vtkGUISupportQt_LOADED)
-  add_subdirectory(EasyView)
-  add_subdirectory(CustomLinkView)
-  add_subdirectory(StatsView)
-endif()
 
 if(vtkInfovisParallel_LOADED)
   add_executable(ParallelBFS ParallelBFS.cxx)
